README.md

```diff
@@ -1,2 +1,26 @@
-# SalesforceMobileSDK-UITests
-UI tests for the Mobile SDK
+[![CircleCI](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-UITests/tree/master.svg?style=svg)](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-UITests/tree/master)
```

Comment:
Main Project Readme.  Things specific to either platform or more in depth can live in Android/README or iOS/README.